internal/cmd/local/local_credentials.go

```diff
@@ -15,32 +16,98 @@ import (
 const (
 	airbyteAuthSecretName = "airbyte-auth-secrets"
 	airbyteNamespace      = "airbyte-abctl"
+
+	secretPassword     = "instance-admin-password"
+	secretClientID     = "instance-admin-client-id"
+	secretClientSecret = "instance-admin-client-secret"
 )
 
 func NewCmdCredentials(provider k8s.Provider) *cobra.Command {
+	spinner := &pterm.DefaultSpinner
+
+	var (
+		flagSetPassword string
+		flagSetEmail    string
+	)
+
 	cmd := &cobra.Command{
 		Use:   "credentials",
 		Short: "Get Airbyte user credentials",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return telClient.Wrap(cmd.Context(), telemetry.Install, func() error {
-
-				client, err := defaultK8s(provider.Kubeconfig, provider.Context)
+			return telClient.Wrap(cmd.Context(), telemetry.Credentials, func() error {
+				k8sClient, err := defaultK8s(provider.Kubeconfig, provider.Context)
 				if err != nil {
 					pterm.Error.Println("No existing cluster found")
 					return nil
 				}
-				secret, err := client.SecretGet(cmd.Context(), airbyteNamespace, airbyteAuthSecretName)
+
+				secret, err := k8sClient.SecretGet(cmd.Context(), airbyteNamespace, airbyteAuthSecretName)
 				if err != nil {
 					return err
 				}
 
-				pterm.Success.Println(fmt.Sprintf("Getting your credentials: %s", secret.Name))
-				pterm.Info.Println(fmt.Sprintf("{\n  \"password\": \"%s\",\n  \"client-id\": \"%s\",\n  \"client-secret\": \"%s\"\n}", secret.Data["instance-admin-password"], secret.Data["instance-admin-client-id"], secret.Data["instance-admin-client-secret"]))
+				clientId := string(secret.Data[secretClientID])
```

Comment:
NIT (iirc): I believe secret has a `.StringData` property that would mean you don't need the cast.

Reply:
From the docs
```
	// stringData allows specifying non-binary secret data in string form.
	// It is provided as a write-only input field for convenience.
	// All keys and values are merged into the data field on write, overwriting any existing values.
	// The stringData field is never output when reading from the API.
```
I don't think it is actually returned from the get/read call.

Reply:
Ah, makes sense I think the context where I used it was in some tests so I may have actually written to it myself